.gitignore

```
@@ -1,6 +1,7 @@
-__pycache__/
+**/__pycache__/
```

Is it the case that `__pycache__/` dirs won't be written to the root dir? Otherwise the wildcard version will skip them so best to keep both. 
```suggestion
__pycache__/
**/__pycache__/
```

---

`**` should also include `.` no? https://gist.github.com/jstnlvns/ebaa046fae16543cc9efc7f24bcd0e31 this page seems to suggest that it matches the root directory as well.

---

Yeah sorry, I just tested it out then and your right.